lib/chat_api_web/router.ex

```diff
@@ -46,12 +46,14 @@ defmodule ChatApiWeb.Router do
 
     get("/slack/oauth", SlackController, :oauth)
     get("/slack/authorization", SlackController, :authorization)
+    put("/widget_configs/create_or_update", WidgetConfigController, :createOrUpdate)
 
     resources("/user_invitations", UserInvitationController, except: [:new, :edit])
     resources("/accounts", AccountController, only: [:update, :delete])
     resources("/messages", MessageController, except: [:new, :edit])
     resources("/conversations", ConversationController, except: [:new, :edit, :create])
     resources("/customers", CustomerController, except: [:new, :edit, :create])
+    resources "/widget_configs", WidgetConfigController, except: [:new, :edit]
```

Comment:
probably don't need to expose every endpoint for the resource here, since all we care about is `create_or_update` and `show` (i.e. `PUT /api/widget_config` with auth and `GET /api/widget_config?account_id=xxxxx`)